docs/devel/contributing.md

```diff
@@ -2,74 +2,91 @@
 
 This guide will guide you through the process of setting up a development environment and contributing to Nuclio. 
 
-## Set up some prerequisites
-Obviously, you'll need:
+#### In This Document
+- [Prerequisites](#prerequisites)
+- [Getting the source code](#get-source)
+- [Setting up a GoLand project](#goland-setup)
+- [Submitting a PR](#submitting-a-pr)
+
+<a id="prerequisites"></a>
+## Prerequisites
+
+Ensure that your setup includes the following prerequisite components:
+
 - Linux or OSX
-- git
-- Docker (version 17.05+, because Nuclio uses [multi-stage builds](https://docs.docker.com/engine/userguide/eng-image/multistage-build/))
+- Git
```

Comment:
```suggestion
- git
```

mainly because it refers to the command line utility

Reply:
It's capitalized in the official Git documentation — https://git-scm.com/.